EIC-EA/EIC-community-D8
# Feature/eicnet group review message

### What I did

before 
![image](https://user-images.githubusercontent.com/15520511/142405972-76ab8ee4-62e3-48ef-a86f-1c57c68e78cf.png)

now
![image](https://user-images.githubusercontent.com/15520511/142406028-5b534590-3599-4547-88f4-3591ff319603.png)

### How to test

- As an Admin, go to public group and block it
- Logout and as an Anonymous go to this group
- Normally you see the message
